client-java/controller/src/main/java/org/evomaster/client/java/controller/problem/rpc/schema/types/JavaDtoSpec.java

```diff
@@ -0,0 +1,16 @@
+package org.evomaster.client.java.controller.problem.rpc.schema.types;
+
+/**
+ * supported specification of Java Dto
+ */
+public enum JavaDtoSpec {
+    /**
+     * pure Java
+     */
+    PURE,
```

Comment:
`PURE` might not be best word here... "pure" functions have a very specific meaning, not sure if confusing then dealing with "pure" dtos.
what about `DEFAULT` ?